linux/ubuntu/scripts/act.sh

```diff
@@ -138,6 +138,25 @@ for ver in "${NODE[@]}"; do
   "${NODEPATH}"/bin/npm -v
 done
 
+case "$(uname -m)" in
+  'aarch64')
+    scripts=(
+      yq
+    )
+    ;;
+  'x86_64')
+    scripts=(
+      yq
+    )
+    ;;
+  *) exit 1 ;;
+esac
+
+for SCRIPT in "${scripts[@]}"; do
+  printf "\n\t🧨 Executing %s.sh 🧨\t\n" "${SCRIPT}"
+  "/imagegeneration/installers/${SCRIPT}.sh"
+done
```

Comment:
this breaks building on any arch other than aarch64/x86_64

Reply:
Yeah good point, I looked at the wrong matrix. The yq script already supports arm/v7.
I'll have a look at updating the cases.

Does this approach otherwise look OK?

Reply:
Added the missing arch